test/uk/gov/hmrc/cgtpropertydisposals/controllers/returns/SubmitReturnsControllerSpec.scala

```diff
@@ -287,59 +239,18 @@ class SubmitReturnsControllerSpec extends ControllerSpec {
 
           val representeeAnswers = sample[CompleteRepresenteeAnswers].copy(id = NoReferenceId)
 
-          val registrationDetails = RegistrationDetails(
-            representeeAnswers.name,
-            representeeAnswers.contactDetails.emailAddress,
-            representeeAnswers.contactDetails.address
-          )
-
-          val sapNumber = sample[SapNumber]
-
-          val subscriptionDetails =
-            SubscriptionDetails(
-              Right(representeeAnswers.name),
-              representeeAnswers.contactDetails.contactName,
-              representeeAnswers.contactDetails.emailAddress,
-              representeeAnswers.contactDetails.address,
-              sapNumber
-            )
-
           val requestBody = sample[SubmitReturnRequest].copy(
             checkYourAnswerPageHtml = B64Html(""),
             completeReturn = sample[CompleteSingleDisposalReturn].copy(
               representeeAnswers = Some(representeeAnswers)
             )
           )
 
-          "the call to register without id fails" in {
-            mockRegisterWithoutId(registrationDetails)(Left(Error("")))
-
+          "the call fails" in {
```

Comment:
isn't this test about when no reference id is provided? The test description could be more accurate here